config/initializers/omniauth.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :facebook, ENV['FACEBOOK_KEY'], ENV['FACEBOOK_SECRET'],
+            scope: 'public_profile', 
+            info_fields: 'id,name,link'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.